share/scripts/inverse/run_gromacs.sh

```diff
@@ -115,12 +117,32 @@ if [[ ${CSG_MDRUN_OPTS} ]]; then
   mdrun_opts+=" ${CSG_MDRUN_OPTS}"
 fi
 
+#support of REMD during gromacs simlation
+if [[ $(csg_get_property cg.inverse.gromacs.REMD) = "yes" ]]; then
+PREFIX="$(csg_get_property cg.inverse.gromacs.REMD.PREFIX)"
+read -a T <<<"$(csg_get_property cg.inverse.gromacs.REMD.T_for_replicas)"
+j=0
+for i in "${T[@]}"
+do
+echo $i
+#TEMP has to be written as a place holer for the temperature in the *.mdp file
+sed "s/TEMP/$i/" grompp.mdp > grompp_$j.mdp
+critical ${grompp[@]} -n "${index}" -f "grompp_$j.mdp" -p "$topol_in" -o "$PREFIX"_"$j.tpr" -c "${conf}" ${grompp_opts} 2>&1 | gromacs_log "${grompp[@]} -n "${index}" -f "${mdp}" -p "$topol_in" -o "$tpr" -c "${conf}" ${grompp_opts}"
```

Comment:
I thought it's more user friendly , if they just have to change one line in the xml file, instead of changing one line in n files (related to the mdp files). But if it's better to parse the files via the list, it's possible to do so